treebeard/models.py

```diff
@@ -85,11 +85,11 @@ def load_bulk(cls, bulk_data, parent=None, keep_ids=False):
             nodes
 
 
-        :param keep_ids:
+        :param keep_pks:
 
-            If enabled, loads the nodes with the same id that are given in the
-            structure. Will error if there are nodes without id info or if the
-            ids are already used.
+            If enabled, loads the nodes with the same pk that are given in the
+            structure. Will error if there are nodes without pk info or if the
+            pks are already used.
```

Comment:
This reads wrong in prose. Can we change it to read "primary key" instead of "pk"?

Reply:
Done